test/e2e/scheduling/predicates.go

```diff
@@ -90,20 +89,6 @@ var _ = SIGDescribe("SchedulerPredicates [Serial]", func() {
 		// Every test case in this suite assumes that cluster add-on pods stay stable and
 		// cannot be run in parallel with any other test that touches Nodes or Pods.
 		// It is so because we need to have precise control on what's running in the cluster.
-		systemPods, err := framework.GetPodsInNamespace(cs, ns, ignoreLabels)
```

Comment:
ignoreLabels declared but will not used.

Reply:
The above comment doesn't make sense anymore when you remove this code, as well.

Reply:
Also, do you have a list of what the pre-condition was waiting for that is missing on openshift?